composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": "^7.1.3",
         "symfony/process": "^4.2",
-        "symfony/console": "dev-master",
+        "symfony/console": "4.3.x-dev",
```

Comment:
It's 4.4 according to the branch of the commit:
https://github.com/symfony/symfony/pull/32806/commits/5f451e6f7074976aae60a65ba144fff684b7b48c

Reply:
Apparent success :)

Reply:
I guess it have been copied/cherry-picked 1 minor down.